fullstack/pom.xml

```diff
@@ -6,119 +6,70 @@
         <artifactId>spring-boot-starter-parent</artifactId>
         <groupId>org.springframework.boot</groupId>
         <version>1.4.1.RELEASE</version>
-        <relativePath/>
+        <relativePath />
     </parent>
 
-    <groupId>org.terracotta.demo</groupId>
-    <artifactId>demo</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
+    <groupId>org.ehcache.sample</groupId>
+    <artifactId>fullstack</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
     <packaging>war</packaging>
-    <name>Demo</name>
+    <name>fullstack</name>
 
     <prerequisites>
         <maven>3.0.0</maven>
     </prerequisites>
 
     <properties>
-        <hikaricp.version>2.4.6</hikaricp.version>
         <assertj-core.version>3.5.2</assertj-core.version>
         <awaitility.version>1.7.0</awaitility.version>
         <commons-io.version>2.5</commons-io.version>
         <commons-lang.version>3.4</commons-lang.version>
         <frontend-maven-plugin.version>1.0</frontend-maven-plugin.version>
-        <gatling.version>2.1.7</gatling.version>
         <gatling-maven-plugin.version>2.1.7</gatling-maven-plugin.version>
+        <gatling.version>2.1.7</gatling.version>
```

Comment:
why reshuffling so many lines in this file ?
history is not readable anymore...